serverless.example.yml

```diff
@@ -26,8 +25,8 @@ provider:
 
 functions:
   wallet-watcher:
-    handler: src/handlers.walletWatcherHandler
-    timeout: 360
-    memorySize: 128
+    handler: ./ncc/index.walletWatcherHandler
+    timeout: 900
```

Comment:
Was it intentional to bump it to 900? 😅 
They could also overlap: 900s = 15 mins, but interval = 10 mins.

Reply:
🕵️ nice catch, not intentional